src/condor_credd/condor_credmon_oauth/credmon/CredentialMonitors/LocalCredmon.py

```diff
@@ -132,6 +132,7 @@ def generate_access_token_info(self, username: str, token_name: str) -> TokenInf
                 except IOError:
                     self.log.exception("Could not open {mapfile}, cannot add group authorizations".format(mapfile=self.authz_group_mapfile))
 
+        profile = ""
         if self.token_ver:
             profile = self.token_ver
```

Comment:
Setting this to an empty string still means that no version claim will be made in the token per lines 187-191:
```python
        # Only set the version if we have one.  No version is valid, and implies scitokens:1.0
        if info.profile == "wlcg:1.0" or info.profile == "wlcg":
            token.update_claims({"wlcg.ver", "1.0"})
        elif info.profile:
            token.update_claims({'ver': info.profile})
```